src/components/NavigatorSidebar.vue

```diff
@@ -100,12 +98,11 @@ import DocumentationNav from 'theme/components/DocumentationTopic/DocumentationN
 const NAVIGATOR_HIDDEN_ON_LARGE_KEY = 'navigator-hidden-large';
 
 export default {
-  name: 'BaseNavigatorView',
+  name: 'NavigatorSidebar',
```

Comment:
Mmm, I'm not happy with the name... "Navigator" and "Sidebar" means the same to me.

Maybe `DocumentationLayout` makes more sense? Essentially this is a layout component that allows any view to have the same look and feel than the documentation pages. Not only that, it allows you to use the `DocumentationTopicStore` and the `DocumentationNav`